dev/create_release.sh

```diff
@@ -98,9 +97,9 @@ function make_binary_release() {
 git_clone
 make_source_package
 
-make_binary_release netinst "-Pweb-angular -Phadoop-2.6 -pl !hbase,!jdbc,!file,!flink,!cassandra,!elasticsearch,!bigquery,!alluxio,!livy,!groovy,!java,!neo4j,!submarine,!sparql,!mongodb -am"
```

Comment:
`hadoop-2.6` also does not take effect. currently, `hadoop-2.7` is the default one

Reply:
an additional question which beyond this PR's scope.

`!flink` was listed here, but it does not work. what's the original intention here? do we want to include flink stuff into `netinst` artifacts?